py3status/modules/getjson.py

```diff
@@ -94,28 +92,23 @@ def _query_url(self):
         """
         """
         try:
-            resp = self.py3.request(self.url, timeout=self.timeout)
-            status = resp.status_code == 200
-            resp = resp.json()
+            resp = self.py3.request(self.url, timeout=self.timeout).json()
```

Comment:
It is possible that `.json` fails here eg set `url = 'google.co.uk'`

I think really the `py3.request` should handle this better and provide an exception that would be caught by the general `py3.RequestException`  but could also be caught explicitly something like `py3.RequestInvalidJSON`